machines/auth.ts

```diff
@@ -149,7 +149,7 @@ export const authMachine = model.createMachine(
       }),
 
       setLanguage: assign({
-        selectLanguage: (context) => !context.selectLanguage,
+        selectLanguage: (context) => true,
```

Comment:
can you please, share why this is reverted, will this invoke the language selector screen all the time ?

Reply:
It won't invoke all the time. It invokes only when the password or biometrics are setup